pkg/test/framework/components/namespace/namespace.go

```diff
@@ -32,6 +32,9 @@ type Config struct {
 	Revision string
 	// Labels to be applied to namespace
 	Labels map[string]string
+	// SkipDump, if enabled, will disable dumping the namespace. This is useful to avoid duplicate
```

Comment:
Is istio-system the main reason for introducing this? Can you expand on why we're dumping istio-system twice? Seems like we should fix that without needing this flag, no?

Reply:
`Istio` component has its own dumping logic which is a superset of the namespace dumping. But since we call ClaimNamespace now we end up dumping the istio-system namespaces.

In theory we could make the Istio component not duplicate the ns dumping parts, but that would mean its in a different folder (harder to debug) and relies on ClaimNamespace being called (doesn't always happen)